CONTRIBUTING.md

```diff
@@ -26,9 +29,63 @@ The Corporate CLA is for those who are contributing as a member of an organizati
 - [CLA for individuals](https://opensource.atlassian.com/individual)
 - [CLA for corporate contributors](https://opensource.atlassian.com/corporate)
 
+## Key Concepts
+
+### CSS-in-JS
+
+Compiled is a CSS-in-JS library: a library that lets React developers write CSS within their React code. Other libraries in this space include styled-components and Emotion. Below are some code examples.
+
+How people usually write in Compiled (we do want to change this long-term). This is similar to the styled function from [styled-components](https://styled-components.com/).
+
+```javascript
+import { styled } from '@compiled/react';
+
+const MyComponent = styled.div({
+  margin: 0,
+  color: 'blue',
+  padding: '250px',
+  backgroundColor: 'pink',
+});
+
+export const App = () => <MyComponent>Hello!</MyComponent>;
+```
+
+However, long term we want developers to write more like below. This is more similar to [emotion](https://emotion.sh/docs/css-prop).
+
+```javascript
+/** @jsx jsx */
+import { css } from '@compiled/react';
+
+const myStyles = css({
+  margin: 0,
+  color: 'blue',
+  padding: '250px',
+  backgroundColor: 'pink',
+});
+
+export const App = () => <div css={myStyles}>Hello!</div>;
+```
+
+### Abstract Syntax Trees
+
+ASTs are a core concept that is used in the babel plugin (bulk of compiled implementation) and automations via codemods or ESLinters. An AST is an abstract representation of the syntax that makes up the programming language. ASTs are used to manipulate JavaScript code at build time, or to automatically refactor code. This concept is important for understanding automation, which is key to maintaining any moderate to large codebase.
```

Comment:
We're already in the key concept section, so it's a bit redundant. Also lead in with the definition first.

```suggestion
An AST is an abstract representation of the syntax that makes up the programming language. ASTs are used to manipulate JavaScript code at build time, or to automatically refactor code. This concept is important for understanding automation, which is key to maintaining any moderate to large codebase.

The `@compiled/babel-plugin` is backed by ASTs and forms the core implementation of Compiled. We recommend using [AST Explorer](https://astexplorer.net/) to gain a working understanding of ASTs, and our codebase.
```